draft-ietf-oauth-transaction-tokens.md

```diff
@@ -349,17 +352,18 @@ JWT claims as well as defines new claims. These claims are described below:
 `sub`:
 : REQUIRED A unique identifier for the subject within the context of the `aud` Trust Domain. Unlike OpenID Connect, the `sub` claim is NOT associated with the `iss` claim.
 
-`purp`:
-: REQUIRED A String defining the purpose or intent of this transaction.
+`scope`:
+`scope`:
+: REQUIRED A JSON string as defined in {{RFC8693}}. Note that the value of this claim is determined by the Transaction Token Service and is not required to match the requested scope nor the scope in any supplied external token.
```

Comment:
```suggestion
: REQUIRED Thee scope claim is defined in {{RFC8693}}. Note that the value of this claim is determined by the Transaction Token Service and is not required to match the requested scope nor the scope in any supplied external token.

Reply:
you should even be able to ref the specific section like this:
```suggestion
: REQUIRED The scope claim is defined in {{Section 4.2 of RFC8693}}. Note that the value of this claim is determined by the Transaction Token Service and is not required to match the requested scope nor the scope in any supplied external token.
```